crates/fuel-core/src/schema/tx/assemble_tx.rs

```diff
@@ -802,7 +831,7 @@ where
             match &status.result {
                 TransactionExecutionResult::Success { .. } => break,
                 TransactionExecutionResult::Failed { receipts, .. } => {
-                    for receipt in receipts.iter().rev() {
+                    for receipt in receipts.iter() {
```

Comment:
i think this is also a breaking change if people are using plain indexes to access some specific receipt, why was this change made?

Reply:
It was added in the parent PR(which is not released yet), and I forgot to remove it there=D I wanted to have an optimization assuming that panic receipts at the end, but we can have multiple panic receipts now, so this optimization will not work because we still need to iterate over all receipts

Reply:
ok sounds good